src/dotnet/Mgmt.CI.BuildTools/ci.common.props

```diff
@@ -11,7 +11,9 @@
 		<OutputPath>$(CIBuildOutputDir)</OutputPath>
 		<SignAssembly>false</SignAssembly>
 		<DelaySign>false</DelaySign>
-		<!-- <AssemblyOriginatorKeyFile>$(RepoToolsDir)\MgmtNetSdkToolsKey.snk</AssemblyOriginatorKeyFile> -->
+		<!-- <AssemblyOriginatorKeyFile>$(RepoToolsDir)\MgmtNetSdkToolsKey.snk</AssemblyOriginatorKeyFile> 
```

Comment:
bad copy/paste?

Reply:
I don't want to sign it and maybe it's a remanant of last time I attempted to sign build tools.
Will remove it.

Reply:
I was actually more wondering about the new change which added netcorepp2.1 as part of the comment and less about why this is commented out. However if it isn't needed it would be good to clean-up.